src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/HttpOntologyDAO.scala

```diff
@@ -0,0 +1,30 @@
+package org.broadinstitute.dsde.firecloud.dataaccess
+
+import akka.actor.ActorSystem
+import org.broadinstitute.dsde.firecloud.model.Ontology.TermResource
+import org.broadinstitute.dsde.firecloud.model.ModelJsonProtocol._
+import org.broadinstitute.dsde.firecloud.utils.RestJsonClient
+import spray.client.pipelining._
+import spray.http.Uri
+import spray.httpx.SprayJsonSupport._
+import spray.httpx.unmarshalling._
+import spray.json.DefaultJsonProtocol._
+
+import scala.concurrent.{ExecutionContext, Future}
+
+
+class HttpOntologyDAO(implicit val system: ActorSystem, implicit val executionContext: ExecutionContext)
+  extends OntologyDAO with RestJsonClient {
+
+  override def search(term: String): Future[Option[List[TermResource]]] = {
+    unAuthedRequest(Get(Uri(ontologySearchUrl).withQuery(("id", term)))) map { response =>
+      if (response.status.isSuccess)
```

Comment:
Not really sure you need to handle the success case - you have either something you can unmarshal, or None. Untested, but I think you can get away with just:
```
        response.entity.as[List[TermResource]] match {
          case Right(obj) => Some(obj)
          case Left(error) => None
        }
```